codespell_lib/data/dictionary.txt

```diff
@@ -20409,6 +20421,14 @@ negatiotiators->negotiators
 negativ->negative
 negatve->negative
 negible->negligible
+negitiable->negotiable
+negitiate->negotiate
+negitiated->negotiated
+negitiates->negotiates
+negitiating->negotiating
+negitiation->negotiation
```

Comment:
```suggestion
negitiation->negotiation
negitiations->negotiations
```

Reply:
Something does go wrong with GitHub after all. I have committed all suggested changes so far. The suggested change seems suspect: what's wrong with line 20249 and why does GitHub suggest a change?

![negitiation](https://user-images.githubusercontent.com/3234522/154293687-d545285e-edd4-4b7c-8fba-939017e7876a.png)
?